protocols/dcutr/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.12.0 -- unreleased
+
+- Implement refactored `Transport`.
+  See [PR 4568](https://github.com/libp2p/rust-libp2p/pull/4568)
```

Comment:
For all protocols, this isn't true because what we are actually changing is the `NetworkBehaviour` interface.

I'd suggest we just make these changelog entries: "Update to `libp2p-swarm` `v0.45.0`.". You can write it as an HTML comment to satisfy the CI rule for needing to update it. Generally, we don't include version bumps in here because they are not useful information to users.

```suggestion
<!-- Update to libp2p-swarm v0.45.0 -->
```